tests/functionaltests/Tests/CortanaTests.mm

```diff
@@ -125,79 +125,79 @@
 // Delegate for testing Cortana foreground activation
 // Use text value to guarantee it is the same argument we create
 @interface CortanaVoiceCommandForegroundTestDelegate : NSObject <UIApplicationDelegate>
-@property (nonatomic, readonly) NSMutableDictionary* methodsCalled;
+@property (nonatomic, readonly) StrongId<NSMutableDictionary> methodsCalled;
```

Comment:
sweet emoji. What part seems non ideal? Having a strongID prop seems fine to me. I don't particularly like the methodsCalled doodad but it works and follows AJs pattern

---
In reply to: [82296469](https://github.com/Microsoft/WinObjC/pull/1100#discussion_r82296469) [](ancestors = 82296469)